apps/token-manager/app/src/hooks.js

```diff
@@ -76,26 +73,20 @@ export function useArrowKeysFocus(query, containerRef = useRef()) {
   )
 
   const { handleBlur: handleContainerBlur } = useOnBlur(reset, containerRef)
-  useEffect(
-    () => {
-      if (highlightedIndex === -1) {
-        return
-      }
-      const elements = document.querySelectorAll(query)
-      if (!elements[highlightedIndex]) {
-        return
-      }
-      elements[highlightedIndex].focus()
-    },
-    [highlightedIndex, query]
-  )
-  useEffect(
-    () => {
-      document.addEventListener('keydown', handleKeyDown)
-      return () => document.removeEventListener('keydown', handleKeyDown)
-    },
-    [handleKeyDown]
-  )
+  useEffect(() => {
```

Comment:
By updating to `_AutoComplete` from `aragonUI` these hooks will become deprecated

Reply:
Done in 1f492cf.